samples/bluetooth/peripheral_nucleo_wb55rg/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _bluetooth-peripheral_nucleo_wb55rg:
+
+Bluetooth: STM32WB55 BLE peripheral demo
+########################################
+
+Overview
+********
+
+Application demonstrating BLE peripheral on `ST Nucleo WB55RG`_
+It has battery service and vendor-specific GATT services that it exposes.
+
+Requirements
+************
+
+* ST BLE Sensor Android app `ST BLE Sensor app`_
+* A board with STM32WB55 like ST Nucleo WB55RG
+* Zephyr kernel environment configured for ST Nucleo WB55RG `ST Nucleo WB55RG port`_
```

Comment:
Since this is a zephyr doc, we can reference it with the label at the top of its .rst file:
```
* :ref:`Zephyr kernel environment configured for ST Nucleo WB55RG <nucleo_wb55rg_board>`
```

Reply:
fixed